### ISSUE TITLE
Allowing only 1 boss to spawn in region

### ISSUE THREAD

**Issue by Blobbyguy** [kangarko/Boss#91]
Hi, I was wondering if there was an efficient way to make it so ONLY the selected boss for a WG region can spawn in it? E.g If it's a zombie boss, only the zombie boss would be able to spawn. No other mobs (or animals) would spawn.

**Comment by kangarko** [comment]
Hello,

Boss support limiting the amount of Bosses in a WorldGuard region. Simply select the region in /boss menu -> Spawning -> Regions -> WorldGuard, right click twice and select the limit:

http://prntscr.com/j6jlbr

**Comment by kangarko** [comment]
You have to, however, disable that other animals can spawn, via the WorldGuard flag manually.

**Comment by Blobbyguy** [comment]
Hi, I'm still having issues with this. 
I've set it up so all mobs that can spawn (Except zombies) are prevented from spawning. I've then set the chance of converting natural spawns to bosses in this world to 100% and then allowed spawning in this specific region. However, they spawn fine for the first little bit after the restart and then they seem to slow down, to the point of not spawning at all. Here are the configs:

https://pastebin.com/LqastjbA
https://pastebin.com/VPp5SUG4

(Note - I enabled them coming out of spawners for testing, but I really want them converting from natural spawns). Thanks for the help in advanced!

**Comment by kangarko** [comment]
Are you saying that the conversion will stop after a while and the Bosses will be lower then 50?

You can also test using Timed_Spawning option, that will work better and gives insurance for spawning whereas converting depends on vanilla behavior which varies.

**Comment by Blobbyguy** [comment]
It's kinda unusual. It's like natural spawns completely stop after a while. I'll give the timed_spawning a look and keep fiddling with the natural spawns. Thanks :)

**Comment by kangarko** [comment]
Yes, do not rely on natural spawning as we cannot warranty its functionality.

I am closing as I assume that the timed spawning solves the issue, however if you still have it please comment below.
